src/rendering/framebufferrenderer.cpp

```diff
@@ -1195,9 +1198,11 @@ void FramebufferRenderer::render(Scene* scene, Camera* camera, float blackoutFac
     }
 
     {
-        TracyGpuZone("Overlay");
-        ghoul::GLDebugGroup group("Overlay");
-        data.renderBinMask = static_cast<int>(Renderable::RenderBin::Overlay);
+        TracyGpuZone("Sticker")
```

Comment:
Would "Final"/"Last" maybe be another naming alternative?  Maybe not as descriptive of the behavior, but very clear that it's the last bin and that it's rendered after everything else

or "OnTop" or something. "Sticker" seems pretty ambiguous to me

Reply:
Let's keep the "Sticker" name for now